packages/client/src/upload.ts

```diff
@@ -86,17 +85,6 @@ export async function* upload(
 
         await semaphore.acquire();
 
-        const fPath = file.names[0];
```

Comment:
The file object already contains the contents as a buffer, so there's no need to read the files again.

Reply:
Was this change necessary for this PR? Might be best to do it in a separate PR because I'm not sure the implications here.

Reply:
https://github.com/vercel/vercel/pull/8366